source/index.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * cuid.js
+ * Collision-resistant UID generator for browsers and node.
+ * Sequential for fast db lookups and recency sorting.
+ * Safe for element IDs and server-side lookups.
+ *
+ * Extracted from CLCTR
+ *
+ * Copyright (c) Eric Elliott 2012
+ * MIT License
+ */
+
+'use strict';
+
+let c = 0;
```

Comment:
Use const for all these... don't use the UGLY_UPPERCASE_CONVENTION. I do everything const by default, and I don't like source files yelling at me. ;)